src/ontology/mondo-edit.obo

```diff
@@ -584963,6 +584963,18 @@ is_a: MONDO:0003767 {source="NCIT:C50888", source="https://orcid.org/0009-0009-6
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8003" xsd:anyURI
 
+[Term]
+id: MONDO:1030010
+name: precerebral artery stenosis
+def: "A disorder characterized by an abnormal narrowing of a precerebral artery." [https://orcid.org/0009-0009-6791-365X]
```

Comment:
Definition:
This definition implies that "any disorder" that has an abnormal narrowing of a precerebral artery is a "precerebral artery stenosis". This is not correct.
The definition (genus-differentia) should use the most specific genus, which here should be "cerebrovascular disorder" (ie the parent term).

dbxref:
this ORCID refers to a private record. Is this the correct orcid? If so, who's orcid is it?
Please find another x-ref to add here. Maybe one of the x-ref would work?

Reply:
updated